lib/Dialect/Torch/Transforms/RecomposeComplexOps.cpp

```diff
@@ -121,6 +121,44 @@ class RecomposeSelectFill_ : public OpRewritePattern<AtenFill_TensorOp> {
     return success();
   }
 };
+
+class RecomposeSplitTensorGetItemOp
```

Comment:
Can you add an e2e test that used to fail and now passes because of this pattern?